packages/base/src/features/OpenUI5Enablement.js

```diff
@@ -0,0 +1,40 @@
+import { registerFeature } from "../FeaturesRegistry.js";
+import BusyIndicatorStyles from "../generated/css/BusyIndicator.css.js";
+
+const getBusyIndicatorStyles = () => {
+	return BusyIndicatorStyles;
+};
+
+const setBusyLit = (html, host, templateResult, forStaticArea) => {
```

Comment:
Do not pass a "forStaticArea" parameter here - move the check outside of this entire function